### PR TITLE
[exporter/coralogix] Update readme

### PR DESCRIPTION
**Pull request by ofirshmuel**
**Description:** <Describe what has changed.>

We update the coralogix exporter readme and add endpoints

**Link to tracking Issue:** <Issue number if applicable>

**Testing:** <Describe what testing was performed and which tests were added.>

**Documentation:** <Describe the documentation added.>

### CODE REVIEW

File: exporter/coralogixexporter/README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Example configuration:
 exporters:
   coralogix:
     # The Coralogix traces ingress endpoint
-    endpoint: "https://api.coralogix.com"
+    endpoint: "api.coralogix.com"
```
Comment (oded-dd) [inline exporter/coralogixexporter/README.md:20]: This is not the tracing endpoint so it should be replaced with real one.

**Review by oded-dd — APPROVED** [review]

File: exporter/coralogixexporter/README.md [hunk 2]
```diff
@@ -50,7 +50,7 @@ Prometheus is currently the leading tool for metric collection, it’s easy to i
 exporters:
   prometheusremotewrite:
     # The Coralogix metrics ingress endpoint
-    endpoint: "https://api.coralogix.com"
+    endpoint: "https://metrics-api.coralogix.com"
```
Comment (jpkrohling) [inline exporter/coralogixexporter/README.md:53]: Interesting, the tracing endpoint doesn't have protocol, but the metrics does?

Comment (ofirshmuel) [reply exporter/coralogixexporter/README.md:53]: Yes, metrics endpoint has "https" protocol but to traces endpoint has no protocol

Comment (jpkrohling) [reply exporter/coralogixexporter/README.md:53]: Thanks for confirming.

**Review by jpkrohling — APPROVED** [review]

**Pull request merged**